experiments/long_runs/figures_function.jl

```diff
@@ -127,3 +127,79 @@ function make_figures(
     end
     return nothing
 end
+
+"""
+    check_conservation(root_path, outdir)
+
+Generates one .pdf file called "conservation_figures.pdf" in the provided 
+root_path.
+
+The resulting .pdf contains a time series of the global mean 
+(area-weighted) energy and water volume error, in units of 
+`J/m^2` and `m`. Only continents are included in the global average. 
```

Comment:
Should we maybe add a reference scale for the quantities being considered here? 

I cannot judge if our conversation is good or bad (with my untrained eyes) because I don't know what to expect as a scale.

Reply:
Hmm yes. really bad would be an error that grows in time. Fractionally our error is <~1e-8 in global mean... But maybe I can convert our plots to show the error as a flux, or something? I'll think about it and address before merging.

Reply:
I added the typical value of the global mean of energy/area and water volume per area to the plots.